apps/dashboard/src/components/workflow-editor/steps/email/maily.tsx

```diff
@@ -132,10 +132,13 @@ export const Maily = (props: MailyProps) => {
                       return dedupAndSortVariables(filteredVariables, queryWithoutSuffix);
                     }
 
+                    const iterableName = editor?.getAttributes('for')?.each;
```

Comment:
for now the iterable name is the full path, e.g. - we can polish this in separate PR, it was too difficult for me to change all that at once

```
for each "payload.comments"
   {{payload.comments.id}}
```